docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS
```

Comment:
this is now called Install .NET Core and how you install is not even mentioned. There's not even a link to the downloads page if you just want to download the runtime/SDK on Windows and install it. Seems very misleading and prone to low CSAT with such a title.